ingestion/src/metadata/ingestion/source/database/databricks/metadata.py

```diff
@@ -112,8 +112,12 @@ class MAP(String):
     }
 )
 
-# This method is from hive dialect originally but
-# is overridden to optimize DESCRIBE query execution
```

Comment:
this has not been resolved, let's revert the comment deletion

Reply:
looks good right?